src/mavlink-router/logendpoint.h

```diff
@@ -17,6 +17,11 @@
  */
 #pragma once
 
+#ifndef _GNU_SOURCE
+#define _GNU_SOURCE // for aio_init
+#endif
```

Comment:
Since config.h is generated, I wasn't sure if it's guaranteed to be defined.

Reply:
it is always defined exactly because them we don't have to add these defines before includes. There isn't a single case I know of _GNU_SOURCE is not what we want.

Reply:
I removed it. I didn't see where it comes from, but I found it now: https://github.com/intel/mavlink-router/blob/master/configure.ac#L13.